server/router/test.js

```diff
@@ -4,7 +4,7 @@ import { useDummyData } from '../../test/mock-data-source';
 
 describe('GET /__health__', () => {
   it('returns 200, to be checked by load balancer', () => {
-    request(app)
+    return request(app)
```

Comment:
Probably less error prone to remove the `{}`s as it's a single expression.

Handling async tests like this seems *really* fragile, easy to accidentally not return anything. Is there a better way to do this?